### PR TITLE
display profile and edit information

### PR DESCRIPTION
**Pull request by HugoRaza**
adding the profile edit and show. directly form the nav bar picture when you are log in.

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -16,6 +16,13 @@
     <%= f.input :password_confirmation,
                 required: true,
                 input_html: { autocomplete: "new-password" } %>
+    <%= f.input :first_name,
```
Comment (leonboutrand) [inline app/views/devise/registrations/new.html.erb]: you should remove these lines mate

File: app/views/vehicles/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<h1>Edit your vehicle</h1>
```
Comment (leonboutrand) [inline app/views/vehicles/edit.html.erb]: don't seems to be changed...

**Review by leonboutrand — CHANGES_REQUESTED** [review]
should remove all the fields in devise files

File: .gitignore [hunk 3]
```diff
@@ -13,3 +13,4 @@ node_modules
 yarn-error.log
 .byebug_history
 .env*
+.env*
```
Comment (leonboutrand) [inline .gitignore]: :'(

Comment (HugoRaza) [reply .gitignore]: Check this out 🗡

**Review by leonboutrand — APPROVED** [review]

**Pull request merged**